docs/zh-CN/sql-manual/sql-reference/Data-Manipulation-Statements/OUTFILE.md

```diff
@@ -75,10 +75,20 @@ INTO OUTFILE "file_path"
         broker.kerberos_principal: 指定 kerberos 的 principal
         broker.kerberos_keytab: 指定 kerberos 的 keytab 文件路径。该文件必须为 Broker 进程所在服务器上的文件的绝对路径。并且可以被 Broker 进程访问
         
-        HDFS 相关属性需加前缀 `hdfs.`:
-        hdfs.fs.defaultFS: namenode 地址和端口
-        hdfs.hdfs_user: hdfs 用户名
-        
+        HDFS 相关属性:
+        fs.defaultFS: namenode 地址和端口
```

Comment:
Why remove `hdfs.` prefix?
Better be compatible with old version.

Reply:
The HDFS parameter attribute names supported by different SQL are not uniform enough, sometime use [hdfs]. as prefix, sometimes use [dfs.] as prefix and sometime use [fs.] as prefix easily confused, just want to normalize them.